backend/Controllers/MetadataController.cs

```diff
@@ -114,6 +114,9 @@ public async Task<IActionResult> PostAsync([FromBody] SaveMetadataResource resou
 			if (!ModelState.IsValid || !Enum.IsDefined(typeof(EReleaseState), resource.ReleaseState))
 				return BadRequest(ModelState.GetErrorMessages());
 
+			if (resource.ReleaseState != EReleaseState.Red && string.IsNullOrEmpty(resource.Url))
```

Comment:
Is this supposed to be:
```
if(resource.ReleaseState == EReleaseState.Released && string.IsNullOrEmpty(resource.Url)
```
?

Reply:
That depends on where we want to require an url.  As it is in the code now I only allow  unreleasable datasets to not have an url.

Reply:
```C#
public enum EReleaseState: byte
	{
		[Description("The data is open")]
		Released = 1,

		[Description("Ready for release")]
		Green = 2,

		[Description("Needs work")]
		Yellow = 3,

		[Description("Not possible to release")]
		Red = 4,
	}
```

Reply:
That is the enum, I interpret it has Released is the only that requires url?

Reply:
I know how the enum looks like. But as i said it depends on where we want to require an url. If we want only released to require an url then the frontend needs to be changed too.

Reply:
@petterroea